packages/react/src/components/Combobox/Combobox.test.tsx

```diff
@@ -515,6 +515,9 @@ test('should prevent default with enter keypress and open listbox', () => {
     // rtl doesn't let us mock preventDefault
     // see: https://github.com/testing-library/react-testing-library/issues/572
     event.preventDefault = preventDefault;
+    if (type === 'focus') {
```

Comment:
The changes here I suspect are related to https://github.com/testing-library/user-event/issues/592, but we're not using `user-event` but are calling `fireEvent` due to specific limitations with testing library.